From de0c128646aabaa4cbba44d9c3d4313412c4913b Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 05 Dec 2018 23:20:09 +0200
Subject: [PATCH] version: move preboot_ver_str under !tester builds

The preboot_ver_str string is only accessed from the preboot_version()
function and thus is useles on TESTER builds.
Move it under !tester builds.

Change-Id: I276e986a0da69265b37ff72ae3d12228d80e1942
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/preboot.c b/preboot.c
index da175de..62ec0a6 100644
--- a/preboot.c
+++ b/preboot.c
@@ -468,9 +468,9 @@
 	preboot_puts("PCIe link is UP!\n");
 }
 
+#ifndef TARGET_IS_TESTER
 static const char *preboot_ver_str = PREBOOT_VER_STR;
 
-#ifndef TARGET_IS_TESTER
 static void preboot_version(void)
 {
 	u64 offset = (u64) preboot_ver_str;
